config/stack.go

```diff
@@ -12,15 +12,7 @@ func Stacks() []string {
 		"osx-vs4mac-beta",
 		"osx-vs4mac-previous-stable",
 		"osx-vs4mac-stable",
-		"osx-xcode-10.1.x",
```

Comment:
Would you mind elaborating why did you remove these?

Reply:
This is a hardcoded list of available stacks, the list has changed: https://github.com/bitrise-io/bitrise.io/tree/master/system_reports

Reply:
Makes sense, thanks for the clarification!